static/js/containers/pages/b2b/B2BPurchasePage_test.js

```diff
@@ -67,7 +67,15 @@ describe("B2BPurchasePage", () => {
     const props = inner.find("B2BPurchaseForm").props()
     assert.deepEqual(
       props.products,
-      products.filter(product => product.visible_in_bulk_form === true)
+      products
+        .filter(product => product.visible_in_bulk_form === true)
+        .sort((a, b) =>
+          a.title.toLowerCase() > b.title.toLowerCase()
+            ? 1
+            : b.title.toLowerCase() > a.title.toLowerCase()
+              ? -1
+              : 0
+        )
     )
   })
 
```

Comment:
@ahmed-arbisoft I agree with @gsidebo , we should do in the api side. Earlier we chose to do sorting because of the older version of python which was not keeping the order intact but now we have upgraded it to `python 3.7`. So it can be done in the backend.

> I tried to fix it in Python which would not compromise scalability. But python does not preserve the dictionary order in less than Python 3.6. Currently we are using Python 3.5.2 for it. So it cannot be utilized, only the frontend option is feasible for now.

> Here is the reference of it.

    https://bugs.python.org/issue38414

Reply:
We did in course run list sorting for `/ecommerce/admin/enroll` if you remember.

Reply:
Updated